models/med_classifer_models.py

```diff
@@ -0,0 +1,209 @@
+"""
```

Comment:
Are we still using this file? I thought we were only using this as reference, correct me if I'm wrong.